models/controllers/helpers.go

```diff
@@ -13,6 +16,16 @@ import (
 	"github.com/spf13/viper"
 )
 
+const BrokerPingEndpoint = "8222/connz"
+
+type Connections struct {
```

Comment:
Where will this struct be used?

Reply:
https://github.com/meshery/meshkit/pull/197/files#diff-a127b332cfc1dd1adf96e28240276fe52634ce187468c511bf31d30d55321a0eR112